### ISSUE TITLE
Experience and Education order should default latest first

### ISSUE THREAD

**Issue by darmenise** [publicv-org/publicv#10]
In the CV both Experience and Education should have a default order. Now the default is `scope :chronological_order, -> { order(created_at: :desc) }` but it should actually show the latest ones on top. Sometimes it can be difficult to decide which is the latest. Consider, for instance:

A) started on 2015 ended on 2020
B) started on 2018 ended on 2020
C) started on 2016 ended on `nil` (still going)
D) started on 2019 ended on `nil` (still going)

The default order should be:

D
C
B
A

In other words, we should look at `ended_on` first, putting the nils on top, and then look at `started_on`.

**Comment by rahul-bluebash** [comment]
@vishal-bluebash PR for this  -  https://github.com/publicv-org/publicv/pull/32
